src/portable/nordic/nrf5x/dcd_nrf5x.c

```diff
@@ -891,6 +895,11 @@ static bool hfclk_running(void)
 
 static void hfclk_enable(void)
 {
+#if CFG_TUSB_OS == OPT_OS_MYNEWT
+  usb_clock_request();
+  return;
+#endif
```

Comment:
we should do #ifelse for the whole hfclk_enable() / hfclk_disable() to prevent gcc to emit `unreachable statement warning`